extensions/binding/org.eclipse.smarthome.binding.mqtt/README.md

```diff
@@ -0,0 +1,97 @@
+# MQTT Broker Connections
+
+    MQTT is a machine-to-machine (M2M)/"Internet of Things" connectivity protocol.
+    It was designed as an extremely lightweight publish/subscribe messaging transport.
+
+This binding allows to list non-mutable system MQTT broker connections for status check purposes as well as configure new broker connections.
+The hereby configured broker connections are shared with the Eclipse Smarthome framework and can be enumerated and consumed by other bindings
+that share the "mqtt" binding namespace.
+
+
+## Supported Bridges
+
+* Broker: This bridge represents a MQTT Broker connection, configured and managed by this binding.
+* SystemBroker: This bridge is a read-only MQTT Broker connection and represents a system configured broker. We are not able to change any of the connections settings of such a broker connection.
+
+## Discovery
+
+There are two discovery mechanism implemented so far.
+One lists the broker connections of the Eclipse Smarthome framework, especially the system configured ones.
```

Comment:
Formulate it towards users, i.e. remove "Eclipse Smarthome framework" and rather say "system broker connections".